src/RA_Dlg_MemBookmark.cpp

```diff
@@ -628,8 +628,8 @@ void Dlg_MemBookmark::WriteFrozenValue(const MemBookmark & Bookmark)
     if (!Bookmark.Frozen())
         return;
 
-    unsigned int addr;
-    unsigned int width;
+    unsigned int addr{};
```

Comment:
There were some warnings about potential uninitialized variables, so these make sure these are initialized to the default value.
value_types and pointers usually aren't automatically default constructible.